x-pack/plugins/security_solution/public/app/translations.ts

```diff
@@ -25,6 +25,13 @@ export const ENTITY_ANALYTICS_RISK_SCORE = i18n.translate(
   }
 );
 
+export const NOTES_MANAGEMENT = i18n.translate(
+  'xpack.securitySolution.navigation.notesManagement',
+  {
+    defaultMessage: 'Notes management',
```

Comment:
nit: `Notes` should be enough since you are already in Management Section.

Also, reminder that it will need to be added to kibana search bar results on the top.